pkg/logging/config_test.go

```diff
@@ -30,27 +30,36 @@ import (
 )
 
 func TestNewLogger(t *testing.T) {
-	logger, _ := NewLogger("", "")
+	logger, _, err := NewLogger("", "")
 	if logger == nil {
 		t.Error("expected a non-nil logger")
 	}
+	if err == nil {
+		t.Error("expected error")
```

Comment:
if you have time, it would be great to see a bit more info in these error messages about what went wrong, e.g. "expected an error when creating a logger with an empty config"